operators/cuda/add_mul.h

```diff
@@ -8,6 +8,35 @@
 
 namespace contrib {
 
+inline void _FillOutputShape3Op(std::vector<int64_t>& dimsA,
```

Comment:
Consider using `const std::vector<int64_t>&` for fixed inputs.

Reply:
None of them are fixed.